server/stycoon/commands.go

```diff
@@ -0,0 +1,49 @@
+package stycoon
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/rs/zerolog/log"
+)
+
+func ParseCommands(s string) (map[string]Command, error) {
+	var c map[string]Command
+	err := json.Unmarshal([]byte(s), &c)
+	return c, err
+}
+
+func (game *Game) ProcessCommands(commands map[string]Command) map[string]string {
+	res := map[string]string{}
+	for strId, command := range commands {
+		id, err := strconv.Atoi(strId)
+		if err != nil {
+			res[strId] = fmt.Sprintf("unable to convert id to number")
```

Comment:
missing continue here?

Reply:
also mention the value in the error - we can return it to the user for simpler debug

Reply:
Added the continue, thanks. The number is already present - it's the map key.